payments/stripe/__init__.py

```diff
@@ -31,6 +42,13 @@ def __init__(self, public_key, secret_key, image="", name="", **kwargs):
         self.image = image
         self.name = name
         super().__init__(**kwargs)
+        warnings.warn(
+            _(
+                "This provider uses the deprecated v2 API, please use `payments.stripe.StripeProviderV3`"
+            ),
```

Comment:
This doesn't need translations; it's not a user-facing string.

```suggestion
            "This provider uses the deprecated v2 API, please use `payments.stripe.StripeProviderV3`",
```